web/static.go

```diff
@@ -121,3 +146,18 @@ func unsupportedBrowserScriptHandler(w http.ResponseWriter, r *http.Request) {
 	templatesDir, _ := templates.GetTemplateDirectory()
 	http.ServeFile(w, r, filepath.Join(templatesDir, "unsupported_browser.js"))
 }
+
+func getOGDescriptionMetaTag(c *Context) string {
```

Comment:
It seems a bit odd that all of the `title` manipulation is in the caller function, then this is in its own function. Maybe we can put all of the calculation of this in its own function, such as `getOpenGraphMetaTags`. Even though one of them is the `title` tag, I think it's ok to have the function named that.

Then the function above can just say

```go
modifiedHTML := getOpenGraphMetaTags(originalHTML)
```

or something like that

Reply:
That's a good point. I updated it now 😃